R/tm_t_events.R

```diff
@@ -459,12 +452,16 @@ template_events <- function(dataname,
   y
 }
 
-#' Teal Module: Events by Term
+#' teal Module: Events by Term
+#'
+#' @description teal module for generating a table of events by term. Please, check out the
+#'  [TLG-catalog](https://insightsengineering.github.io/tlg-catalog/stable/) for
+#'  additional examples and information.
```

Comment:
I added in the tlg-c link like this (only to the modules that are used somewhere in the catalog)
```suggestion
#' @seealso The [TLG Catalog](https://insightsengineering.github.io/tlg-catalog/stable/) where additional example
#'   apps implementing this module can be found.
```